### ISSUE TITLE
Accessibility Violations: Hyperlinks must contain link text 

### ISSUE THREAD

**Issue by myan9** [IBM/kui#2656]
This issue covers: https://github.com/IBM/kui/issues/2507#issuecomment-528475529
